backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -95,12 +95,14 @@ class CardMutationService {
         }
 
         if (!overwrite && card.totpSecret != null) {
+            logger.info("Card with id:${card.id} did not overwrite card from ${context.remoteIp}")
             return CardActivationResultModel(ActivationState.did_not_overwrite_existing)
         }
 
         val totpSecret = CardActivator.generateTotpSecret()
         val encodedTotpSecret = Base64.getEncoder().encodeToString(totpSecret)
         transaction { CardRepository.activate(card, totpSecret) }
+        logger.info("Card with id:${card.id} was activated from ${context.remoteIp}")
```

Comment:
What's the reason for logging the IP addresses? I think, we don't really need them for debugging purposes. It might be more interesting whether `overwrite` is true or not at this point.

Reply:
The main reason to log the ip adress can be to block certain ip adresses if they spam our server. I think for the activation this could only make sense for failed activations, but feel free to disagree :)

Reply:
i added the overwrite value